src/js/components/Widgets/EndorsementCard.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
```

Comment:
For a simple component like this, you can use `PureComponent` instead of `Component` so that it doesn't re-render each time new (but identical) props get passed in.